dist/s_executable

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# cd into the directory u want
```

Comment:
np you*

Reply:
oops forgot to remove my comments, fixed https://github.com/wiredtiger/wiredtiger/pull/10063/commits/11f569d291f2fe26dd6ddf8fe76a81a7150e02a6